### ISSUE TITLE
Granular testing and representative sampling

### ISSUE THREAD

**Issue by workday-accessibility** [w3c/silver#365]
One main challenge we see in rolling WCAG 3 out though is that many of the tests and outcomes focus on atomic level details which are very granular. This level of granularity makes measuring compliance more difficult for large applications. We would like WCAG 3 to include a standard methodology for how to best approach these granular tests in larger applications once the guidelines further develop; for example, there should be a best practice recommendation around how to conduct representative sampling for accessibility in large applications, and how to represent usage metrics in connection to why representative sampling was conducted in a particular way.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
